lektor/packages.py

```diff
@@ -75,10 +75,8 @@ def remove_package_from_project(project, name):
     return None
 
 
-def download_and_install_package(
-    package_root, package=None, version=None, requirements_file=None
-):
-    """This downloads and installs a specific version of a package."""
+def download_and_install_packages(package_root, packages_with_versions):
```

Comment:
I'd suggest the name `requirements` (or perhaps `requirement_specifiers`), rather than `packages_with_versions` since that what pip [calls them](https://pip.pypa.io/en/stable/reference/pip_install/#requirement-specifiers).

Reply:
[x] done